src/HDInsight/HDInsightOnAks.Autorest/test/sessionRecords/ClusterOperations.Recording.json

```diff
@@ -0,0 +1,3137 @@
+{
+  "ClusterOperations+[NoContext]+[NoScenario]+$PUT+https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/PStestGroup/providers/Microsoft.HDInsight/clusterpools/ps-test-cluster-pool?api-version=2023-06-01-preview+1": {
+    "Request": {
+      "Method": "PUT",
+      "RequestUri": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/PStestGroup/providers/Microsoft.HDInsight/clusterpools/ps-test-cluster-pool?api-version=2023-06-01-preview",
```

Comment:
I'm suspecting that putting json files under a subfolder instead of test folder directly blocks tests runs in playback mode. Have you tried on local ./test-module.ps1 -playback

Reply:
It can work because we specify the recording.json file location in the related test ps1 file.
The result of `./test-module.ps1 -playback` is as below:
![Uploading image.png…]()